examples/convert_mri_and_trans.py

```diff
@@ -24,19 +24,145 @@
 See the documentation pages in the MNE docs for more information on
 `source alignment and coordinate frames <mne_source_coords_>`_
 
-
-
 """
 # Authors: Stefan Appelhoff <stefan.appelhoff@mailbox.org>
 # License: BSD (3-clause)
 
 ###############################################################################
 # We are importing everything we need for this example:
-#
-#
-#
-#
+
+import os
+import os.path as op
+import shutil as sh
+
+import numpy as np
+import mne
+from mne.datasets import sample
+from mne.transforms import apply_trans
+import nibabel as nib
+import matplotlib.pyplot as plt
+from nilearn.plotting import plot_anat
+
+from mne_bids import (write_raw_bids, make_bids_basename, write_anat,
+                      get_head_mri_trans)
+from mne_bids.utils import print_dir_tree, _extract_landmarks
```

Comment:
you should not expose a private function in an example

Reply:
+1

Reply:
Then how else should I get the coordinates?

- paste the actual code from the private function in there?
- make a new public function?
- ... ?